src/routes/routes.js

```diff
@@ -1,12 +1,37 @@
 import express from "express";
+import { getAllAssignment } from "../controllers/assignment.controller.js";
 import {
+  createClass,
+  deleteClass,
+  getClass,
+  getClassName,
+  getStudentList,
+} from "../controllers/class.controller.js";
+import {
+  createResource,
+  getAllResource,
+} from "../controllers/resource.controller.js";
+import {
+  createSchoolController,
   getAllSchoolController,
   getSchoolController,
 } from "../controllers/school.controller.js";
 
 const routes = express.Router();
 
 routes.route("/schools").get(getAllSchoolController);
-routes.route("/school/:id").get(getSchoolController);
+routes.route("/school/:sid").get(getSchoolController);
+routes.route("/create-school").post(createSchoolController);
+
+routes.route("/resources").post(createResource);
```

Comment:
Instead of `/create-school` can we use just `/school` ?
Just for consistency as other routes have!!